### ISSUE TITLE
Import image improvements

### ISSUE THREAD

**Issue by macumber** [NatLabRockies/floorspace.js#260]
- [x] Provide warning if image can't be imported (e.g. select a pdf)
- [x] What does z do?  
- [x] Is image opacity working?  I can enter values of -100, 0, 0.1, 0.99, 1, 100, 1000 and I don't see any difference after importing png
- [x] Can you no longer move an image?

**Comment by bgschiller** [comment]
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image),
> The only image formats SVG software must support are JPEG, PNG, and other SVG files. Animated GIF behavior is undefined.

I checked .gif and it worked, so I allowed it. .tiff did not.

`z` is intended to control image layering -- which one is on top. It's not implemented right now, so I hid the column.

Image opacity is now working (in my branch).

Image moving is also working in the branch.
